core/src/main/boilerplate/sttp/tapir/internal/ParamsToSeq.scala.template

```diff
@@ -5,7 +5,6 @@ private[tapir] object ParamsToSeq {
     a match {
       [2..#case ([#v1#]) => Seq([#v1#])#
       ]
-      case ()                       => Seq()
```

Comment:
I'm surprised this doesn't break any tests ;) can you add a test using `in_unit_out_json_unit` to `ClientTests` to verify that this works as well?

Reply:
also, this makes it not symmetrical with `SeqToParams`. Not sure if there are some implications ... maybe not :)

Reply:
It doesn't 

> I'm surprised this doesn't break any tests ;) can you add a test using `in_unit_out_json_unit` to `ClientTests` to verify that this works as well?

It doesn't, because shrot-cuircuting is added in [`EncodeOutputs`](https://github.com/softwaremill/tapir/pull/500/files#diff-e54e326061b36dd199c50c742895b905R16). IMO this line was inconsistent with the next one - all single values like `v1: String` ended up as `Seqv1: String)`, whereas unit specifically turned into empty `Seq`.

Client test for `in_unit_out_json_unit` is now added.

> also, this makes it not symmetrical with SeqToParams. Not sure if there are some implications ... maybe not :)

I've tried to make them symmetric, but it's not for free - it would require some refactoring of `EndpointToSttpClient`, that I don't believe to bring any additional value. Are you good with leaving `SeqToParams` as is?

Reply:
yes, that should be fine. As long as all tests pass, we should be good :)